packages/browser/tsconfig.json

```diff
@@ -5,5 +5,6 @@
 
   "compilerOptions": {
     // package-specific options
+    "declarationDir": "build/types"
```

Comment:
```suggestion
```

I also realized that this doesn't generate types once, it generates types twice, with one set overwriting the other. (This tsconfig is used by both the cjs and esm builds.)